src/components/results/doctorSpecialtiesListItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class DoctorSpecialtiesListItem extends React.Component {
+  
+    render() {
+        console.log(this.props.specialties);
```

Comment:
[fix] Calling this property "specialties" is confusing, since it's only one specialty. I had to double-check that you weren't passing an array here.